src/main/webapp/WEB-INF/tags/instructor/moreInfo.tag

```diff
@@ -0,0 +1,36 @@
+<%@ tag description="instructorCourseStudentDetails / instructorStudentRecords - More Info Modal & Panel" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ attribute name="student" type="teammates.ui.template.StudentProfile" required="true" %>
+<%@ tag import="teammates.common.util.Const" %>
+<c:set var="notFilled"><i class="text-muted"><%= Const.STUDENT_PROFILE_FIELD_NOT_FILLED %></i></c:set>
+<div class="modal fade" id="studentProfileMoreInfo" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+    <div class="modal-dialog modal-lg">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                <h4 class="modal-title">${student.name}'s Profile - More Info</h4>
+            </div>
+            <div class="modal-body">
+                <br>
+                <!--Note: When an element has class text-preserve-space, do not insert and HTML spaces-->
+                <p class="text-preserve-space height-fixed-md">${empty student.moreInfo ? notFilled : student.moreInfo}</p>
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-primary" data-dismiss="modal">Close</button>
+            </div>
+        </div><!-- /.modal-content -->
+    </div><!-- /.modal-dialog -->
+</div><!-- /.modal -->
```

Comment:
Don't think these explanations (`modal-content`/`dialog`/...) are necessary anymore since now we can see clearly which closing div belongs to which. But if you want to retain it go ahead.